src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/ReleaseServicePermissionTests.cs

```diff
@@ -179,8 +180,13 @@ await PolicyCheckBuilder<SecurityPolicies>()
                 .AssertForbidden(
                     userService =>
                     {
-                        var service = BuildReleaseService(userService: userService.Object);
-                        return service.CreateReleaseAmendmentAsync(_release.Id);
+                        using (var contentDbContext = InMemoryApplicationDbContext("CreateReleaseAmendmentAsync"))
```

Comment:
Technically don't need the curly braces after `userService =>` but I'm just nit-picking :)

Reply:
I cannot remove it here so left it as it is